Packages/manifest.json

```diff
@@ -9,13 +9,13 @@
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.learn.iet-framework": "2.2.1",
     "com.unity.memoryprofiler": "0.5.0-preview.1",
-    "com.unity.netcode.gameobjects": "1.0.0-pre.10",
-    "com.unity.multiplayer.tools": "1.0.0-pre.7",
+    "com.unity.multiplayer.tools": "1.0.0-pre.8",
+    "com.unity.netcode.gameobjects": "1.0.0",
     "com.unity.postprocessing": "3.2.1",
     "com.unity.render-pipelines.universal": "12.1.6",
-    "com.unity.services.authentication": "1.0.0-pre.4",
+    "com.unity.services.authentication": "2.1.1",
     "com.unity.services.lobby": "1.0.0-pre.6",
```

Comment:
lobby should have its own non pre version as well

Reply:
yeah, but as discussed, lobby has a lot of changes in that new version, including deprecating and replacing the Lobbies class that we are currently using, so this will require more work. I'm creating a ticket for investigating this